src/bundle/Resources/public/scss/_pagination.scss

```diff
@@ -2,6 +2,18 @@
     font-size: $ez-pagination-font-size;
 }
 
+.ez-pagination-text {
+    font-size: .75rem;
+}
+
+.ez-pagination-btn {
```

Comment:
The next thing is, you're not following BEM naming convention here.
The selectors in this file should look like:
```scss
.ez-pagination__btn {}
.ez-pagination__spacing {}
.ez-pagination__text {}
```